package.json

```diff
@@ -95,7 +95,7 @@
     "build:manifest": "node bin/substitute-manifest.js",
     "build:intent-metadata": "node bin/parse-intent-toml.js",
     "build:ppn-listing": "node bin/generate-ppn-listing.js > extension/js/vendor/ppnListing.js",
-    "build:jsx": "babel --relative --out-dir . --ignore gh-pages '**/*.jsx'",
+    "build:jsx": "./node_modules/.bin/babel --relative  --out-dir ./ --only '**/*.jsx' extension/ --verbose",
```

Comment:
In package.json the `$PATH` is updated to include `./node_modules/.bin` so you don't need the full path here. The full path is for use on the command line when you want to test things out. Similarly `--verbose` is for testing.